packages/components/src/animate/README.md

```diff
@@ -0,0 +1,39 @@
+# Animate
+
+Simple interface to introduce animations to components. 
+
+## Usage
+
+```jsx
+import { Animate } from '@wordpress/components';
+
+const MyAnimatedNotice = () => (
+	<Animate todo="Add missing props">
+		{ ( { className } ) => (
+			<Notice className={ className } status="success">
+				<p>Animation finished.</p>
+			</Notice>
+		) }
+	</Animate>
+);
+```
+
+## Props
+
+Name | Type | Default | Description
+--- | --- | --- | ---
+`type` | `string` | `undefined` | Type of the animation to use.
+`options` | `object` | `{}` | Options of the chosen animation.
+`children` | `function` | `undefined` | A callback receiving a list of props ( `className` ) to apply to the DOM element to animate.
```

Comment:
When `children` isn't a function it will explode at the moment.

Reply:
I think it's fine, it's a requirement for the component. it's like saying the `options` is an object and if you pass a number it will explode.

I do see the specificity in JSX about the `children` prop but ultimately I think we can just consider it a requirement to be a function.

Reply:
I think it's a conceptual issue with render props :)